docker/gcp_pudl_etl.sh

```diff
@@ -48,7 +48,7 @@ function run_pudl_etl() {
 }
 
 function shutdown_vm() {
-    upload_file_to_slack "$LOGFILE" "pudl_etl logs for $BUILD_ID:"
+    # upload_file_to_slack "$LOGFILE" "pudl_etl logs for $BUILD_ID:"
```

Comment:
Temporarily commented out, because we're not actually running the build below and thus there's no logfile.